lib/__tests__/commonOpts.test.ts

```diff
@@ -1,15 +1,36 @@
-// @ts-nocheck
-const {
+import {
   CMS_PUBLISH_MODE,
   DEFAULT_CMS_PUBLISH_MODE,
-} = require('@hubspot/local-dev-lib/constants/files');
-const {
+} from '@hubspot/local-dev-lib/constants/files';
+import {
   getAndLoadConfigIfNeeded,
   getAccountId,
   getAccountConfig,
   loadConfigFromEnvironment,
-} = require('@hubspot/local-dev-lib/config');
-const { getCmsPublishMode } = require('../commonOpts');
+} from '@hubspot/local-dev-lib/config';
+import { getCmsPublishMode } from '../commonOpts';
+import { CmsPublishMode } from '@hubspot/local-dev-lib/types/Files';
+import { Arguments } from 'yargs';
+
+const mockedGetAccountId = getAccountId as jest.Mock;
+const mockedGetAccountConfig = getAccountConfig as jest.Mock;
+const mockedGetAndLoadConfigIfNeeded = getAndLoadConfigIfNeeded as jest.Mock;
+const mockedLoadConfigFromEnvironment = loadConfigFromEnvironment as jest.Mock;
+
+type CmsPublishModeArgs = {
+  cmsPublishMode?: CmsPublishMode;
+  account?: number | string;
+};
+
+function buildArguments(
```

Comment:
Just a note, but I had to add something similar in my other PR. We may eventually want a util that does this so we don't have to do this everywhere. I can look into that in my PR after this is merged